trillian_api.proto

```diff
@@ -169,7 +179,7 @@ message GetEntryAndProofResponse {
 // Clients cannot directly modify the log data via this API.
 service TrillianLog {
     // QueueLeaf adds a single leaf to the queue.
-    rpc QueueLeaf (QueueLeafRequest) returns (google.protobuf.Empty) {}
+    rpc QueueLeaf (QueueLeafRequest) returns (QueueLeafResponse) {}
```

Comment:
We're going end up reflecting possibly large amounts of data to the client (non CT applications), which does not seem efficient.